src/frontend/Komponenter/Behandling/Brev/Brevmeny.tsx

```diff
@@ -80,7 +80,8 @@ const Brevmeny: React.FC<BrevmenyProps> = (props) => {
     }, []);
 
     useEffect(() => {
-        hentBeløpsperioder(vedtaksresultat);
+        if (vedtaksresultat !== EBehandlingResultat.SANKSJONERE)
+            hentBeløpsperioder(vedtaksresultat);
```

Comment:
Burde kanskej ha med curly brackets? 
```suggestion
        if (vedtaksresultat !== EBehandlingResultat.SANKSJONERE) {
            hentBeløpsperioder(vedtaksresultat);
        }
```

Reply:
Ups